new-log-viewer/src/components/Layout.tsx

```diff
@@ -223,6 +263,14 @@ const Layout = () => {
             default: break;
         }
     }, []);
+    const handleButtonClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+        const actionName = event.currentTarget.getAttribute("data-action-name") as ACTION_NAME;
+        if (Object.values(ACTION_NAME).includes(actionName)) {
+            handleAction(actionName);
+        }
+    };
+    const [open, setOpen] = React.useState<boolean>(false);
```

Comment:
> ESLint: Caution: `React` also has a named export `useState`. Check if you meant to write `import {useState} from 'react'` instead.(import/no-named-as-default-member)

```suggestion
    const [open, setOpen] = useState<boolean>(false);
```

Let's try to fix any lint errors before pushing to PRs now we have the ESLint automation pipeline enabled.